IBPSA/Resources/C-Sources/WeeklySchedule.c

```diff
@@ -501,10 +437,14 @@ void weeklyScheduleFreeInit(void * ID) {
 void weeklyScheduleFree(void * ID) {
   WeeklySchedule* scheduleID = (WeeklySchedule*)ID;
 
+  if (ID == NULL) /* Otherwise OM segfaults when IBPSA.Utilities.IO.Files.Examples.WeeklySchedule triggers an error */
+    return;
+
   int i;
```

Comment:
`int i` must be declared on top of the function according to C90.

Reply:
fixed